dev/com.ibm.ws.ssl/src/com/ibm/ws/ssl/core/WSX509KeyManager.java

```diff
@@ -343,18 +343,26 @@ public String chooseEngineClientAlias(String[] keyType, Principal[] issuers, SSL
         if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled())
             Tr.entry(tc, "chooseEngineClientAlias", new Object[] { keyType, issuers, engine });
 
-        String rc = null;
+        String alias = null;
         if (null != customKM && customKM instanceof X509ExtendedKeyManager) {
             if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled())
                 Tr.debug(tc, "chooseEngineClientAlias, using customKM -> " + customKM.getClass().getName());
-            rc = ((X509ExtendedKeyManager) customKM).chooseEngineClientAlias(keyType, issuers, engine);
-        } else {
-            rc = chooseClientAlias(keyType[0], issuers);
-        }
+            alias = ((X509ExtendedKeyManager) customKM).chooseEngineClientAlias(keyType, issuers, engine);
+        } 
+        //Despite the Javadoc indicating support for passing an array of keyTypes, 
```

Comment:
Looks like you are missing the else here?

Reply:
added else as suggested by Ut. Thanks!